iOS Example Frame/iOS Example Frame/MainViewController.swift

```diff
@@ -50,12 +50,13 @@ final class MainViewController: UIViewController, CardViewControllerDelegate, Th
 
         let cardFormData = Self.defaultCardFormData()
 
-      let paymentFormViewController = PaymentFormFactory.getPaymentFormViewController(
-        environment: environment,
-        billingFormData: cardFormData.billingForm,
-        paymentFormStyle: cardFormData.paymentFormStyle,
-        billingFormStyle: cardFormData.billingFormStyle,
-        supportedSchemes: [.visa, .mastercard, .maestro])
+        let paymentConfiguration = PaymentFormConfiguration(apiKey: "pk_test_6e40a700-d563-43cd-89d0-f9bb17d35e73",
```

Comment:
For reference, this key is already hardcoded in the object on line 25 (no code changes to have it appear). As documented in PR comment, this is not currently used and was the result of my misguided initial approach to the logging.

I do however think the task where this is consumed will show up very soon as we need the network client before this is completed.